### ISSUE TITLE
UI - Home page layout

### ISSUE THREAD

**Issue by RobinaSanderson** [AtlasOfLivingAustralia/profile-hub#518]
This story is still to be confirmed - do not start until this notice is removed.

**Banner changes**
The banner for the home page will include:
- the text "Flora of Australia online" on a white background
- which blends into 
- a selection of images which change (image slider where the previous image fades into the next)
This may be achieved by either:
- allowing the administrator to enter text in a standard font to appear on the white background, and allowing the administrator to upload or link to a selection of images, and the system will blend the 2 components of the banner.
- allowing the administrator to upload a series of single images that has already blended the text and image components (note this will require further testing to ensure there are not problems when scaling in cutting off text, it also assumes that the blended components will occupy the same rations in the banner so that the when one image fades into another, the text stays in the same place).

**Buttons, explanatory text and updates**
- These components will occupy 3 columns under the banner
- Background will be white

**Header Menu** 
The header will have a gray background (same gray as the footer)
The header will contain the following options:
- Search (either the search box or a link to the search page (not both) see XXX for information about search)
- Browse - link to browse page
- Glossary - opens a new tab with the glossary displayed
- About - drop down includes Reports
- My profile
- Login/Logout
- help icon

![image](https://cloud.githubusercontent.com/assets/23368496/25930623/87b37568-364a-11e7-9250-dda4ebf22cb6.png)

**Comment by m-hope** [comment]
Superseded by #523
